pkg/scheduler/internal/cache/cache.go

```diff
@@ -348,8 +348,10 @@ func (cache *schedulerCache) ForgetPod(pod *v1.Pod) error {
 		}
 		delete(cache.assumedPods, key)
 		delete(cache.podStates, key)
+	case !ok && !cache.assumedPods[key]:
+		return nil
 	default:
-		return fmt.Errorf("pod %v wasn't assumed so cannot be forgotten", key)
+		return fmt.Errorf("pod %v could not found in podStates or assumedPods", key)
```

Comment:
Either write "could not be found" or change the sentence around to, "could not find pod ..."